ably/options.go

```diff
@@ -443,6 +440,22 @@ func (opts *clientOptions) idempotentRESTPublishing() bool {
 	return opts.IdempotentRESTPublishing
 }
 
+func isActiveInternetConnection() bool {
+	res, err := http.Get(internetCheckUrl)
```

Comment:
We should always use the HTTP client from the ClientOptions.

Reply:
I see this is fixed in the next PR: https://github.com/ably/ably-go/pull/359/files#diff-721ab670921034c0c26f7999ad630004f73001cb85be84c768e7bf93c6abf14fR444